storage/container_storage_interface/persistent-storage-csi-manila.adoc

```diff
@@ -0,0 +1,26 @@
+[id="persistent-storage-csi-manila"]
+= OpenStack Manila CSI Driver Operator
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-manila
+
+toc::[]
+
+== Overview
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for the link:https://wiki.openstack.org/wiki/Manila[OpenStack Manila] shared file system service.
+
+Familiarity with link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/[PVs], link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims[persistent volume claims (PVCs)], link:http://blog.kubernetes.io/2016/10/dynamic-provisioning-and-storage-in-kubernetes.html[dynamic
+provisioning], and link:https://kubernetes.io/docs/admin/authorization/rbac/[RBAC authorization] is recommended.
+
+Before PVCs can be created, you must install the Manila CSI Driver Operator. The Operator creates the required storage classes for all available Manila share types needed for dynamic provisioning.
+
+After the Operator is installed, you must also create the ManilaDriver Custom Resource (CR) that is required in the {product-title} cluster.
+
```

Comment:
We should cover operator uninstall little bit and add a comment that, it is not recommended to uninstall the operator while there are PVs that are using it.

Reply:
Thanks, @gnufied - I was wondering about that. I've added a section on uninstalling the operator that is similar to what we are using for AWS EBS operator uninstall: https://github.com/openshift/openshift-docs/pull/22199/files#diff-07baabcc25ef2ac37420e3e62bc64080